frontend/src/assets/theme/button.scss

```diff
@@ -0,0 +1,6 @@
+// cf: login.component.html
+.btn-success.p-inputtext {
+  background-color: #28a745;
+  border-color: #28a745;
+  color: white;
+}
```

Comment:
I don't knwo where you will go when all components will be on NGprime, so we can begin to divide css rules ?

So if we want to make a border-radius to all buttons, we do it here